tests/unit/test_config.py

```diff
@@ -232,30 +198,6 @@ def test_gateway_listen_multiple_addresses(self):
         ]
         # take the first value
         assert edge_port == 9999
-        assert edge_port_http == 0
-        assert edge_bind_host == "0.0.0.0"
-
-    def test_legacy_variables_override_if_given(self, default_ip):
```

Comment:
Could we reinstate this test but invert the logic, i.e. that if `GATEWAY_LISTEN` is _not_ specified, that setting `EDGE_PORT` does not overwrite it?

Reply:
nice idea 💡 
Done